plugins/repository-s3/src/test/java/org/opensearch/repositories/s3/RepositoryCredentialsTests.java

```diff
@@ -65,17 +65,24 @@
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.atomic.AtomicReference;
 
-import static org.opensearch.repositories.s3.S3ClientSettings.ACCESS_KEY_SETTING;
-import static org.opensearch.repositories.s3.S3ClientSettings.SECRET_KEY_SETTING;
-import static org.opensearch.test.hamcrest.OpenSearchAssertions.assertAcked;
 import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.instanceOf;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.not;
 import static org.hamcrest.Matchers.notNullValue;
+import static org.opensearch.repositories.s3.S3ClientSettings.ACCESS_KEY_SETTING;
+import static org.opensearch.repositories.s3.S3ClientSettings.SECRET_KEY_SETTING;
+import static org.opensearch.test.hamcrest.OpenSearchAssertions.assertAcked;
 
 @SuppressForbidden(reason = "test requires to set a System property to allow insecure settings when running in IDE")
-public class RepositoryCredentialsTests extends OpenSearchSingleNodeTestCase {
+public class RepositoryCredentialsTests extends OpenSearchSingleNodeTestCase implements ConfigPathSupport {
+
+    // @Override
+    // @Before
+    // public void setUp() throws Exception {
+    // SocketAccess.doPrivileged(() -> System.setProperty("opensearch.path.conf", configPath().toString()));
+    // super.setUp();
+    // }
```

Comment:
Put this back and remove from individual tests below?

Reply:
Adding this setUp method here invokes `OpenSearchTestCase.before`, which causes the `threadContext` to record additional warnings. That's why I removed this and added individual lines in each test.

Reply:
Got it. Let's delete the commented out code then.